auth/set_org.go

```diff
@@ -36,5 +35,7 @@ func (s *SetOrgCmd) Run(ctx context.Context, command string) error {
 		return err
 	}
 
-	return login(ctx, cfg, loadingRules.GetDefaultFilename(), s.Organization, runExecPlugin(s.ExecPlugin), project(s.Organization))
+	userInfo := &api.UserInfo{}
```

Comment:
A placeholder for the future. There is a bug in the current implementation of the `set-org` command (nctl auth set-org overwrites kubeconfig with OIDC login) and this block of code will likely be redesigned.